otx/cli/tools/build.py

```diff
@@ -37,11 +35,35 @@ def set_workspace(task, model):
     return path
 
 
-def parse_args():
+def get_args():
     """Parses command line arguments."""
+    # TODO: Declaring pre_parser to get the template
+    pre_parser = argparse.ArgumentParser(add_help=False)
+    pre_parser.add_argument("template", nargs="?", default=None)
+    parsed, _ = pre_parser.parse_known_args()
+    template = parsed.template
     parser = argparse.ArgumentParser()
```

Comment:
I think code L41 ~ L49 is duplicated across many files in otx/cli/tools. How about making these part as a function (manybe in the otx/utils/parser,py?) and using it?

Reply:
reflected this. -> parser.py https://github.com/openvinotoolkit/training_extensions/pull/1641/commits/3fcf067f3ae5f70826b0df66bdb702db0a04c358